src/common/serverinfo.js

```diff
@@ -49,6 +49,12 @@ function getServerInfo(connection: Connection): Promise<GetServerInfoResponse> {
       reserveIncXrp: 'reserveIncrementXRP',
       seq: 'ledgerVersion'
     });
+    info.validatedLedger.baseFeeXRP =
+      info.validatedLedger.baseFeeXRP.toString();
```

Comment:
Argh, my bad. But it is still number in `get-server-info.json`, and test didn't catch this :disappointed: